commands/droplets.go

```diff
@@ -81,6 +82,9 @@ If you do not specify a region, the Droplet is created in the default region for
 	AddStringFlag(cmdDropletCreate, doctl.ArgSizeSlug, "", "", "A `slug` indicating the Droplet's number of vCPUs, RAM, and disk size. For example, `s-1vcpu-1gb` specifies a Droplet with one vCPU and 1 GiB of RAM. The disk size is defined by the slug's plan. Run `doctl compute size list` for a list of valid size slugs and their disk sizes.",
 		requiredOpt())
 	AddBoolFlag(cmdDropletCreate, doctl.ArgBackups, "", false, "Enables backups for the Droplet. By default, backups are created on a daily basis.")
+	AddStringFlag(cmdDropletCreate, doctl.ArgDropletBackupPolicy, "", "",
+		`Path to a droplet backup policy in JSON or YAML format. Set to "-" to read from stdin. Backups must be enabled for the Droplet to apply the backup policy. 
+	Example: doctl compute droplet create my-droplet --size s-1vcpu-1gb --image ubuntu-20-04-x64 --region nyc1 --enable-backups --backup-policy path-to-policy/policy-cfg.yaml`)
```

Comment:
We should avoid requiring a file when possible. As there are only three possible settings here, lets just add some additional flags. Something like:

- `--backup-policy-plan`
- `--backup-policy-weekday`
- `--backup-policy-hour`

Reply:
Done in [4f6dfbd](https://github.com/digitalocean/doctl/pull/1609/commits/4f6dfbde3d120dd11df6311f8ab1ce978a98ecec)